icons/keyboard-off.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "Diottodev",
+    "karsa-mistmere"
+  ],
+  "tags": [
+    "unkeys"
+  ],
+  "categories": [
+    "devices"
```

Comment:
Same here, the categories should be based on categories from the `keyboard`  icon.